.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+## Description
+<!-- Describe your changes in detail -->
+
+## Motivation and context
+<!-- Why is this change required? What problem does it solve? -->
+<!-- If it fixes an open issue, please link the issue here. -->
+
+## How has this been tested?
+<!-- Please describe how you tested your changes. -->
+
+## Screenshots (if appropriate)
+
+## Types of changes
+<!-- What types of changes does your code introduce? -->
+<!-- Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing
+functionality to not work as expected)
+
+## Checklist
+<!-- See the following points, and put an `x` in all the boxes that apply. -->
+<!-- If you're unsure about any of these, please ask. We're here to help! -->
+- [ ] PR title provides summary of the changes and starts with one of the
+[pre-defined prefixes](../../utils/release.yml)
+<!-- For example: "db.describe: add example to documentation" -->
```

Comment:
```suggestion
<!-- For example: "doc: add example to db.describe documentation" -->
<!-- or if it is a fix "db.describe: fix JSON output format" -->
```
Example title suggestions following the current guideline

Reply:
Or `doc: Add example to db.describe documentation` (capital letter)